pagopa/io-app
# chore: [IOBP-578] Added dynamic brand logo from nw onboarding and payment flows

## Short description
This PR adds the possibility to insert the remote icon of any payment method during the onboarding and the payment

## List of changes proposed in this pull request
- Upgraded to the io-app-design-system latest version with the adjustments on the ListItem;
- Added the `paymentLogoUri` and `uri` props to the new list item

## How to test
- Checkout https://github.com/pagopa/io-dev-api-server/pull/354
- Open the onboarding process and you should be able to see all the icons of the onboardable payment methods
- Open the payment flow and during the pick payment method screen, you should be able to see all the right icons for each list item
